navpatch/repositories/fs.go

```diff
@@ -0,0 +1,75 @@
+package repositories
```

Comment:
the goal is keep isolated the several implementation of the same thing in a package. And avoid file names as `repository_fs.go`